app/src/main/java/co/kr/mvvm_study/view/MainActivity.kt

```diff
@@ -24,12 +27,21 @@ class MainActivity : AppCompatActivity() {
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
+        initViewModel()
         initRecyclerView()
         initAddButton()
     }
 
+    private fun initViewModel() {
+        toDoViewModel = ViewModelProvider.AndroidViewModelFactory.getInstance(application)
+            .create(ToDoViewModel::class.java)
+        toDoViewModel.getToDoList().observe(this, {
+            toDoListAdapter.setToDoItems(it)
+        })
```

Comment:
코틀린의 경우 매개변수에 고차함수가 오는 경우

```suggestion
       toDoViewModel.getToDoList().observe(this) {
             toDoListAdapter.setToDoItems(it)
         }
```
이렇게만 써도 됨

Reply:
오 ~